providers/src/airflow/providers/cncf/kubernetes/decorators/kubernetes.py

```diff
@@ -65,7 +65,7 @@ class _KubernetesDecoratedOperator(DecoratedOperator, KubernetesPodOperator):
     # there are some cases we can't deepcopy the objects (e.g protobuf).
     shallow_copy_attrs: Sequence[str] = ("python_callable",)
 
-    def __init__(self, namespace: str = "default", use_dill: bool = False, **kwargs) -> None:
+    def __init__(self, namespace: str | None = None, use_dill: bool = False, **kwargs) -> None:
```

Comment:
do we consider this breaking change or a bug fix?

Reply:
Looks like a breaking change to me, it will affect users who do not initialise the decorator namespace right?

Reply:
Added a warning in changelog and bump the provider version.

Reply:
Thank you